stdlib/_typeshed/__init__.pyi

```diff
@@ -19,8 +19,9 @@ _T = TypeVar("_T")
 _T_co = TypeVar("_T_co", covariant=True)
 _T_contra = TypeVar("_T_contra", contravariant=True)
 
-# Use for "self" annotations:
-#   def __enter__(self: Self) -> Self: ...
+# Alternative to `typing_extensions.Self`, for use with `__new__`:
```

Comment:
I believe the only cases where we still use this are the `__new__` methods of metaclasses, since the Self PEP specifically says that usage of typing.Self is prohibited in metaclasses